core/src/UserSettingsEditor.html

```diff
@@ -1,210 +1,219 @@
+    document
+      .querySelectorAll('.enum-buttons-container-' + key + ' button')
+      .forEach(button => {
+        const buttonId = button.getAttribute('id');
+        const optionId = `lui-us-enum_button_${key}_option`;
+        buttonId === optionId
+          ? button.classList.add('fd-button--emphasized')
+          : button.classList.remove('fd-button--emphasized');
+      });
+
+    storedUserSettingData[userSettingGroup[0]][key] = option;
+  }
 </script>
 
 <style>
-    .lui-form-grid {
-        display: grid;
-        display: -ms-grid;
-        grid-template-columns: minmax(0, 1fr) 2fr;
-        -ms-grid-columns: minmax(0, 1fr) 2fr;
-    }
-
-    .lui-value-container {
-        grid-column: 2;
-        -ms-grid-column: 2;
-    }
-
-    .lui-value-container>input[type="checkbox"] {
-        margin-top: 13px;
-    }
-
-    .lui-form-label {
-        grid-column: 1;
-        -ms-grid-column: 1;
-        text-align: right;
-        padding-top: 10px;
-        overflow: hidden;
-        min-width: 0;
-        word-break: break-word;
-        padding: 10px;
-    }
-
-    .fd-form-label {
-        white-space: pre-line;
-    }
-
-    .lui-us-content-header-container {
-        width: 100%;
-        padding: 1.5rem 2rem;
-        display: flex;
-    }
-
-    .lui-title--h2 {
-        font-size: var(--sapFontHeader2Size, 1.5rem);
-    }
-
-    .lui-title {
-        line-height: var(--sapContent_LineHeight, 1.4);
-        font-family: var(--sapFontFamily, "72", "72full", Arial, Helvetica, sans-serif);
-        font-weight: 400;
-        -webkit-font-smoothing: antialiased;
-        box-sizing: border-box;
-        padding: 0;
-        margin: 0;
-        border: 0;
-        color: var(--sapTextColor, #32363a);
-    }
-
-    .lui-fd-enum-button {
-        margin-top: 6px;
-    }
+  .lui-form-grid {
+    display: grid;
+    display: -ms-grid;
+    grid-template-columns: minmax(0, 1fr) 2fr;
+    -ms-grid-columns: minmax(0, 1fr) 2fr;
+  }
+
+  .lui-value-container {
+    grid-column: 2;
+    -ms-grid-column: 2;
+    position: relative;
+  }
+
+  .lui-value-container>input[type="checkbox"] {
+    margin-top: 13px;
+  }
+
+  .lui-form-label {
+    grid-column: 1;
+    -ms-grid-column: 1;
+    text-align: right;
+    padding-top: 10px;
+    overflow: hidden;
+    min-width: 0;
+    word-break: break-word;
+    padding: 10px;
+  }
+
+  .fd-form-label {
+    white-space: pre-line;
+  }
+
+  .lui-us-content-header-container {
+    width: 100%;
+    padding: 1.5rem 2rem;
+    display: flex;
+  }
+
+  .lui-title--h2 {
+    font-size: var(--sapFontHeader2Size, 1.5rem);
+  }
+
+  .lui-title {
+    line-height: var(--sapContent_LineHeight, 1.4);
+    font-family: var(--sapFontFamily, "72", "72full", Arial, Helvetica, sans-serif);
+    font-weight: 400;
+    -webkit-font-smoothing: antialiased;
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+    border: 0;
+    color: var(--sapTextColor, #32363a);
+  }
```

Comment:
```suggestion

```

Reply:
I couldn't find where it is used in the file

Reply:
👍  thanks. left over